### PR TITLE
AST-1438 - E2E test case for off canvas flyout header type position

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for off canvas flyout header type position

Steps :
1. Go to the customizer and select tablet mode
2. Click on header section and select off-canvas option.
3. Select header type as flyout from general setting
4. In general setting apply the position
5. Check on front end

### Screenshots

https://share.getcloudapp.com/Kou7RDlB

### How has this been tested?

Run single test : npm run test:e2e:interactive tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-position.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-position.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-position.test.js:1]: need to update util path for publish-post

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-position.test.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { setCustomize } from '../../../../../utils/customize';
+import { setBrowserViewport } from '../../../../../utils/set-browser-viewport';
+describe( 'off canvas flyout header type settings in the customizer', () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-position.test.js:4]:
```suggestion
describe( 'Off canvas flyout header type settings in the customizer', () => {
```

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-position.test.js (hunk 1 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-position.test.js]: resolved

File: tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-position.test.js (hunk 2 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/mobile-header/off-canvas-header/flyout-position.test.js]: Resolved

**Review by bsfsnehal — APPROVED** [review]

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**